lib/quicly.c

```diff
@@ -2456,10 +2491,10 @@ static int commit_send_packet(quicly_conn_t *conn, quicly_send_context_t *s, int
     while (s->dst - s->dst_payload_from < QUICLY_MAX_PN_SIZE - QUICLY_SEND_PN_SIZE)
         *s->dst++ = QUICLY_FRAME_TYPE_PADDING;
 
-    /* the last packet of first-flight datagrams is padded to become 1280 bytes */
+    /* the last packet of the first-flight datagrams are padded to become max_udp_payload_size bytes */
```

Comment:
```suggestion
    /* the last packet of the first-flight datagrams is padded to become max_udp_payload_size bytes */
```

Reply:
I think "is" is correct, because the subject (the last packet) is singular?

Reply:
Yup, agreed. I was suggesting changing it back to "is".

Reply:
Oh I missed the context. Thank you for the fix. Applied.